libse/Forms/FixCommonErrors/FixContinuationStyle.cs

```diff
@@ -49,30 +46,24 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
                     if (!ContinuationUtilities.IsEndOfSentence(text))
                     {
                         // ...ignore inserts
-                        if (Configuration.Settings.General.FixContinuationStyleUncheckInsertsAllCaps)
+                        if (Configuration.Settings.General.FixContinuationStyleUncheckInsertsAllCaps
+                            && (ContinuationUtilities.IsAllCaps(text) || ContinuationUtilities.IsAllCaps(textNext)))
                         {
-                            if (ContinuationUtilities.IsAllCaps(text) || (!ContinuationUtilities.IsAllCaps(text) && ContinuationUtilities.IsAllCaps(textNext)))
```

Comment:
`if (ContinuationUtilities.IsAllCaps(text) || (!ContinuationUtilities.IsAllCaps(text) && ContinuationUtilities.IsAllCaps(textNext)))`

`!ContinuationUtilities.IsAllCaps(text)` is always true